INSTALL

```diff
@@ -1,7 +1,7 @@
 # vim: syntax=pod
 
 If you read this file _as_is_, just ignore the funny characters you see.
-It is written in the POD format (see F<pod/perlpod.pod>) which is specially
+It is written in the POD format (see pod/perlpod.pod), which is specially
```

Comment:
The change is good, but "see the file pod/perlpod.pod" might make it clearer. (Do we need to unwrap it further to accommodate users familiar only with OS's that structure paths differently?)

Reply:
I think making this "see the file ..." is reasonable.

I don't think we need to handle OS's that structure paths differently.